import-service/handlers/importFileParser.ts

```diff
@@ -0,0 +1,18 @@
+import { S3Event } from "aws-lambda";
+import { S3 } from "aws-sdk";
+import "source-map-support/register";
+
+import { readS3Stream } from "./parserUtil";
+import { DEFAULT_REGION } from "./../shared";
+
+export const importFileParser = (event: S3Event) => {
```

Comment:
This lambda is non-async, so it will work as expected. According to the [AWS docs](https://docs.aws.amazon.com/lambda/latest/dg/nodejs-handler.html):
`For non-async handlers, function execution continues until the event loop is empty or the function times out. The response isn't sent to the invoker until all event loop tasks are finished.` So there is no need to wait until the end of  s3 stream. Lambda function execution will continue til the end.